lib/new_relic/agent/error_filter.rb

```diff
@@ -144,8 +145,16 @@ def log_filter(setting, errors)
         end
       end
 
+      #edit this to account for integers
       def parse_status_codes(codes)
-        code_list = codes.is_a?(String) ? codes.split(',') : codes
+        code_list = case codes
+          when String
+            codes.split(',')
+          when Integer
+            [codes]
+          else
+            codes
+          end
```

Comment:
Ahhh, I really like the look of this solution. I remember our struggle during pairing with this yesterday. This solution is very easy to read. Nice work!